src/app/components/client/ProgressCard.module.scss

```
@@ -78,6 +80,21 @@
     .progressItem {
       display: flex;
       flex-direction: column;
+      align-items: center;
+      text-align: center;
+      position: relative;
+
+      &:not(:first-child)::before {
```

I had to inspect the code to find this out :)

```suggestion
      // Lines in between the different items in the progress card
      &:not(:first-child)::before {
```